src/AutoRest.CSharp/LowLevel/AutoRest/LowLevelTarget.cs

```diff
@@ -40,6 +41,10 @@ public static async Task ExecuteAsync(GeneratedCodeWorkspace project, InputNames
                 lowLevelClientWriter.WriteClient();
                 project.AddGeneratedFile($"{client.Type.Name}.cs", codeWriter.ToString());
                 project.AddGeneratedDocFile($"Docs/{client.Type.Name}.xml", xmlDocWriter.ToString());
+
+                var exampleCompileCheckWriter = new ExampleCompileCheckWriter(client);
+                exampleCompileCheckWriter.Write();
+                project.AddGeneratedFile($"../../tests/Generated/Samples/Samples_{client.Type.Name}.cs", exampleCompileCheckWriter.ToString());
```

Comment:
In each SDK, there are manually written samples under `tests\Samples`, Shall we put the auto-generated samples under `tests\Samples\Generated` ? In that way, all the samples (manual or auto) are in the same directory.

Reply:
Our convention for generated files is always `subDir/Generated`